### PR TITLE
Vm related

### PR DESCRIPTION
**Pull request by SuperFola**
- Using a unique_ptr<> wrapper on the VM stack to avoid having a 300'000B object
- Exposing the VM Value to the user (it was already exposed through an ugly alias)

### CODE REVIEW

File: include/Ark/Compiler/inline/Compiler.inl [hunk 1]
```diff
@@ -29,7 +29,7 @@ inline std::optional<std::size_t> Compiler::isOperator(const std::string& name)
 inline std::optional<std::size_t> Compiler::isBuiltin(const std::string& name) noexcept
 {
     auto it = std::find_if(internal::Builtins::builtins.begin(), internal::Builtins::builtins.end(),
-        [&name](const std::pair<std::string, internal::Value>& element) -> bool {
+        [&name](const std::pair<std::string, Value>& element) -> bool {
```
Comment (PierrePharel) [inline include/Ark/Compiler/inline/Compiler.inl:32]: you have finally remove Value from internal namespace...

File: include/Ark/VM/Value.hpp [hunk 2]
```diff
@@ -200,7 +197,7 @@ namespace Ark::internal
          * 
          * @param value 
          */
-        explicit Value(Closure&& value) noexcept;
+        explicit Value(internal::Closure&& value) noexcept;
```
Comment (PierrePharel) [inline include/Ark/VM/Value.hpp:200]: of course, Closure too...

**Comment by PierrePharel** [comment]
you have finally remove Value from internal namespace...

**Review by PierrePharel — APPROVED** [review]

**Comment by SuperFola** [comment]
Yup, much better to expose the value (no more proxy class)

**Pull request merged**